### ISSUE TITLE
Add schema support for `ArrayField`

### ISSUE THREAD

**Issue by ghost** [bhch/django-jsonform#72]
Looks like it's not implemented right now. Would love to have this supported. Thoughts?

**Comment by bhch** [comment]
I'll have to think about this.

Meanwhile, you can override the schema using a custom form:

```python
# models.py

class MyModel(...):
    my_array = ArrayField(...)


# forms.py

class MyModelForm(forms.ModelForm):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

        # Provide custom schema
        self.fields['my_array'].widget.schema = {'type': 'array', ...}


# admin.py
class MyAdmin(admin.ModelAdmin):
    form = MyModelForm

...
```

**Comment by dongyuzheng** [comment]
I am @ghost

**Comment by bhch** [comment]
Released in [v2.23.0](https://github.com/bhch/django-jsonform/releases/tag/v2.23.0).
